core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -35,11 +36,18 @@ class ListOffsetsRequestTest extends BaseRequestTest {
 
   val topic = "topic"
   val partition = new TopicPartition(topic, 0)
+  var enableUnstableVersions = true
+
+  @BeforeEach
+  override def setUp(testInfo: TestInfo): Unit = {
+    enableUnstableVersions = !testInfo.getDisplayName.contains("testResponseIncludesLeaderEpochWithUnstableAPIs")
```

Comment:
If the test name contains "UnstableAPI", we set `enableUnstableVersions` to false. Setting it to true will be more intuitive.

Reply:
Yes, It make sence when test is`testResponseIncludesLeaderEpochWithStableAPIs` set `enableUnstableVersions` to false is more better.